lightning: refine description about --check-requirements *bool* (#14238)
pingcap/docs: tidb-lightning/tidb-lightning-command-line-full.md

| `--enable-checkpoint <bool>` | Whether to enable checkpoints (default = true) | `checkpoint.enable` |
| `--analyze <level>` | Analyze tables after importing. Available values are "required", "optional" (default value), and "off". | `post-restore.analyze` |
| `--checksum <level>` | Compare checksum after importing. Available values are "required" (default value), "optional", and "off". | `post-restore.checksum` |

> | `--check-requirements <bool>` | Check cluster version compatibility before starting the task, and check and check whether TiKV has more than 10% free space left after startup. (default = true) | `lightning.check-requirements` |

Author: ```suggestion
| `--check-requirements <bool>` | Check cluster version compatibility before starting the task, and check whether TiKV has more than 10% free space left during running. (default = true) | `lightning.check-requirements` |
```

> | `--check-requirements <bool>` | Check cluster version compatibility before starting the task, and check that TiKV has more than 10% of its available storage space remaining during the running time. (default = true) | `lightning.check-requirements` |

Author: ```suggestion
| `--check-requirements <bool>` | Check cluster version compatibility before starting the task, and check and check whether TiKV has more than 10% free space left after startup. (default = true) | `lightning.check-requirements` |
```
